From 917a82b9fd93d6c994c9bac6ea1deeaefeadd6f3 Mon Sep 17 00:00:00 2001
From: Kevin Mihelich <kevin@archlinuxarm.org>
Date: Tue, 2 Feb 2021 13:58:59 -0700
Subject: [PATCH 3/3] Run blink bindings generation single threaded

When not single threaded this process will eat all the RAM.
---
 chromium/third_party/blink/renderer/bindings/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromium/third_party/blink/renderer/bindings/BUILD.gn b/chromium/third_party/blink/renderer/bindings/BUILD.gn
index 61c9e7f59da..a33c9e4144d 100644
--- a/chromium/third_party/blink/renderer/bindings/BUILD.gn
+++ b/chromium/third_party/blink/renderer/bindings/BUILD.gn
@@ -178,6 +178,7 @@ template("generate_bindings") {
     outputs = invoker.outputs
 
     args = [
+             "--single_process",
              "--web_idl_database",
              rebase_path(web_idl_database, root_build_dir),
              "--root_src_dir",
-- 
2.33.0

